development/nginx/command

```diff
@@ -1 +1,7 @@
+data='{"alert": false, "summary": "convert2rhel did not detect issues", "report": "", "report_json": {"foo": "bar"}}'
+
 /usr/bin/convert2rhel --help
+echo "BEGIN MARKER"
+echo "$data"
+echo "END MARKER"
+/usr/bin/convert2rhel --version
```

Comment:
With these changes, the file sent to upload service : [K7RmkocOjj-000001.txt](https://github.com/oamg/rhc-worker-bash/files/11914934/K7RmkocOjj-000001.txt)